src/SFA.DAS.AssessorService.Application.Api.UnitTests/Controllers/Standards/StandardVersionControllerTests.cs

```diff
@@ -175,14 +176,17 @@ private StandardVersion ConvertFromStandard(Standard standard)
         {
             return new StandardVersion
             {
-                EffectiveFrom = standard.EffectiveFrom.GetValueOrDefault(),
-                EffectiveTo = standard.EffectiveTo,
+                StandardUId = standard.StandardUId,
+                Title = standard.Title,
+                Version = standard.Version.VersionToString(),
                 IFateReferenceNumber = standard.IfateReferenceNumber,
                 LarsCode = standard.LarsCode,
                 Level = standard.Level,
-                StandardUId = standard.StandardUId,
-                Title = standard.Title,
-                Version = standard.Version.ToString(),
+                EffectiveFrom = standard.EffectiveFrom.GetValueOrDefault(),
+                EffectiveTo = standard.EffectiveTo,
```

Comment:
I thought we had standardised this so that EffectiveFrom / EffectiveTo were either both nullable, or not?

Reply:
Fixed